include/links-abstract-domain.php

```diff
@@ -93,4 +96,21 @@ public function set_language_home_urls( $additional_data, $language ) {
 		$additional_data['home_url']   = $additional_data['search_url'];
 		return $additional_data;
 	}
+
+	/**
+	 * Returns language home URL property according to the current domain.
+	 *
+	 * @since 3.4.4
+	 *
+	 * @param string             $url      Home URL.
+	 * @param PLL_Language|array $language Language object or array of language props.
+	 * @return string Filtered home URL.
+	 */
+	public function set_language_home_url( $url, $language ) {
```

Comment:
I wonder if `PLL_Links_Abstract_Domain::set_language_home_urls()` is useful now, since `set_language_home_url()` and `set_language_search_url()` are used in it in `PLL_Links_Model` 🤔

Reply:
I kinda agree with you, the only advantage I see is that we have only one callback for `pll_language_data` filter.
But I propose to handle it in another PR.